packages/next/server/web/sandbox/sandbox.ts

```diff
@@ -243,8 +243,8 @@ function getFetchHeaders(middleware: string, init: RequestInit) {
   return headers
 }
 
-function getFetchURL(input: RequestInfo, headers: NodeHeaders = {}): string {
-  const initurl = isRequestLike(input) ? input.url : input
+function getFetchURL(input: unknown, headers: NodeHeaders = {}): string {
```

Comment:
I think typing `unknown` is not a good idea. If the type is wider than it is stated, I'm sure we can figure out a better type than unknown even though the implementation can remain the same.

Reply:
This function's typings do not propagate to users. `unknown` accurately presents the actual type that can be passed in.

Reply:
That being said it doesn't care that much. I've reverted this internal function's `input` type.